wwwroot/data/regionMapping.json

```diff
@@ -2447,27 +2444,6 @@
             ],
             "description": "Stats New Zealand Mesh Blocks 2017",
             "regionIdsFile": "data/regionids/region_map-NZ_MB2017_HD_Clipped_MB2017.json"
-        },
-        "AEC_FED_2017_AMLS": {
```

Comment:
Did you mean to remove this region type? To my knowledge we still display layers that rely on this

Reply:
No, I didn't. I think this PR is pretty much redundant once https://github.com/TerriaJS/terriajs/pull/3178 goes through though, so I might just close it unmerged.